lib/chef/provisioning/aws_driver/aws_taggable.rb

```diff
@@ -0,0 +1,18 @@
+module Chef::Provisioning::AWSDriver
+# This module is meant to be included in a resource that is taggable
+# This will add the appropriate attribute that can be converged by the provider
+# TODO it would be nice to not have two seperate modules (taggable/tagger)
+#   and just have the provider decorate the resource or vice versa.  Complicated
+#   by resources <-> providers being many-to-many.
+module AWSTaggable
```

Comment:
Seems perfectly reasonable that something marking itself `AWSTaggable` should thereby get `aws_tags`.